### PR TITLE
Addition of tooltip and restore copy functionality

### LINKED ISSUE #2877: Add Tooltip for Event Bar for Mobile View
**Is your feature request related to a problem? Please describe.**

There is no description for `Event Bar` in mobile view, We only get to see `Icons` which are not clear about the appearance or functionality

**Describe the solution you'd like**

Add Tooltip

#### Screenshot
Screenshot 2019-05-08 at 11 01 41 PM

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Fixes #2877 

#### Changes proposed in this pull request:

- Added Tooltips
- Restored copy functionality in mobile view

#### Screenshot

Screenshot 2019-05-08 at 11 27 26 PM

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 Review Please

File: app/templates/events/view.hbs [hunk 1]
```diff
@@ -20,14 +20,22 @@
...
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip="Preview" data-position="top left">
               <i class="unhide icon"></i>
             </a>
-            <button class="ui button" {{action 'togglePublishState'}}>
-              <i class="{{if (eq model.state 'published') 'ban' 'check'}} icon"></i>
+              {{#if (eq model.state 'published')}}
+              <button class="ui button" {{action 'togglePublishState'}} data-tooltip="Unpublish" data-position="top left">
```
Comment (uds5501) [inline app/templates/events/view.hbs]: This if can be condensed into 2 seperate inline ifs
in line 27, use 
`data-tooltip = " {{if isEventPublished 'Unpublish' 'Publish' }} "`
define a component property isEventPublished in the js file.

In line 28, use `<i class = "{{if isEventPublished 'ban' 'check' }} icon"`

File: app/templates/events/view.hbs [hunk 2]
```diff
@@ -20,14 +20,22 @@
...
             </button>
-            <button class="ui button"><i class="copy icon"></i></button>
-            <button class="ui red button" {{action 'openDeleteEventModal'}}><i class="trash icon"></i></button>
+              {{else}}
+              <button class="ui button" {{action 'togglePublishState'}} data-tooltip="Publish" data-position="top left">
+                <i class="check icon"></i>
+                </button>
+              {{/if}}
+            
+            <button class="ui button {{if isCopying 'loading'}}" {{action 'copyEvent'}} disabled="{{isCopying}}" data-tooltip="Copy" data-position="top left">
```
Comment (uds5501) [inline app/templates/events/view.hbs]: Copy event isn't working in my local system. Please confirm.

Comment (kushthedude) [reply app/templates/events/view.hbs]: Wait will check

Comment (kushthedude) [reply app/templates/events/view.hbs]: @uds5501 That is the bug from server side , Events having Tickets in them are not getting copied and Tickets with External link are getting copied . Needs to be handle on different PR

**Comment by kushthedude** [comment]
@CosmicCoder96 Review Please

File: app/templates/events/view.hbs (hunk 1 above)
Comment (uds5501) [reply app/templates/events/view.hbs]: @kushthedude I don't see these changes?

**Comment by kushthedude** [comment]
Forgot to push commit oops

**Comment by kushthedude** [comment]
@CosmicCoder96 Review it please

File: app/templates/events/view.hbs [hunk 3]
```diff
@@ -20,14 +20,16 @@
       <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip="Preview" data-position="top left">
```
Comment (CosmicCoder96) [inline app/templates/events/view.hbs]: use translation helpers for all the tool-tips.

Comment (kushthedude) [reply app/templates/events/view.hbs]: @CosmicCoder96 Done the changes , Review now

File: app/templates/events/view.hbs [hunk 4]
```diff
@@ -20,14 +20,16 @@
...
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip={{t 'Preview'}} data-position="top left">
               <i class="unhide icon"></i>
             </a>
-            <button class="ui button" {{action 'togglePublishState'}}>
+            <button class="ui button" {{action 'togglePublishState'}} data-tooltip="{{if (eq model.state 'published') 'Unpublish' 'Publish'}}" data-position="top left">
               <i class="{{if (eq model.state 'published') 'ban' 'check'}} icon"></i>
             </button>
-            <button class="ui button"><i class="copy icon"></i></button>
-            <button class="ui red button" {{action 'openDeleteEventModal'}}><i class="trash icon"></i></button>
+            <button class="ui button {{if isCopying 'loading'}}" {{action 'copyEvent'}} disabled="{{isCopying}}" data-tooltip={{t 'Copy'}} data-position="top left">
```
Comment (CosmicCoder96) [inline app/templates/events/view.hbs]: if a button is in loadign state, it is disabled by default. No need to make it explicit.

Comment (kushthedude) [reply app/templates/events/view.hbs]: @CosmicCoder96 Review it

**Pull request merged**